src/System.Management.Automation/help/MUIFileSearcher.cs

```diff
@@ -113,25 +114,42 @@ private void SearchForFiles()
             }
         }
 
-        private string[] GetFiles(string path, string pattern)
-        {
+		private string[] GetFiles(string path, string pattern)
+		{ 
```

Comment:
The formatting is fixed. It shows normal under IDE, I have to delete and re tab to fix it.